packages/core/src/external/carequality/pd/process-inbound-pd.ts

```diff
@@ -45,7 +46,7 @@ export async function processInboundPatientDiscovery(
 ): Promise<InboundPatientDiscoveryResp> {
   try {
     const patient = validateFHIRAndExtractPatient(payload);
-    const matchingPatient = await mpi.findMatchingPatient(patient);
+    const matchingPatient = await executeWithNetworkRetries(() => mpi.findMatchingPatient(patient));
```

Comment:
I think we should wrap this in the lower level `findBySimilarityAcrossAllCxs` function within findMatchingPatient as opposed to wrapping it here. 

As a general rule, I think if we wrap it anywhere but in the immediate place where the external service is called, we will have situations where we are accidentally double wrapping and retrying more than we intend to

Reply:
yeah, I agree we should use the retries on the lowest level calls! Will update those!